.github/workflows/linux_scipy.yml

```diff
@@ -43,7 +55,10 @@ jobs:
           #cache-write: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
       - name: Build SciPy
         working-directory: ./scipy
-        run: pixi run build -j2
```

Comment:
Is there a reason why `-j2` was here @rgommers ?

Reply:
`build` on the default environment just went down from 16min to 10min

Reply:
That looks like a copy-paste error. GitHub Actions runners have only 2 cores so you need `-j2` all over the place (`ninja`'s default of `2*n_cpu + 2` is worse), but the Cirrus runner is much more powerful (https://cirrus-runners.app/pricing/)

Reply:
> and was breaking the editable build.

Ah, I see the problem here - `build-editable` is hooked up to `spin install` rather than `spin build`, and the former is missing a bunch of arguments. As a result, we're passing them straight through to `pip` with `spin install -- -Cxxx`, and hence the provided `-j2` ends up behind the `--` and `pip` chokes on it. `--help` also doesn't work, for the same reason.

It's only fixable by adding `--build-dir` and `-C` to the `spin install` command inside `spin` itself, so we can stop using `--`.